docs/get_started/get_started_demos.md

```diff
@@ -249,12 +249,12 @@ Create an `<ir_dir>` directory to contain the model's Intermediate Representatio
 
 @endsphinxdirective
 
-The OpenVINO Runtime can infer models where floating-point weights are [compressed to FP16](../MO_DG/prepare_model/FP16_Compression.md). To generate an IR with a specific precision, run the Model Optimizer with the appropriate `--data_type` option.
+The OpenVINO Runtime can infer models where floating-point weights are [compressed to FP16](../MO_DG/prepare_model/FP16_Compression.md). To generate an IR with FP16 weights, run the Model Optimizer with the `--compress_to_fp16` option.
```

Comment:
The _only_ missing thing is the explanation of WHY the user may want to infer models with compressed weights. I'm not sure about the reading flow, but I believe we are mentioning this capability for the first time here, so better to write 'to save disk space for your IR file you can apply weights compression to FP16...' and so on. But can you even better think about whether it is a good place here to mention it at all?

Reply:
Yes, it is the first mentioning of fp16 compression in get_started_demos quide. Added the note you suggested here.